### ISSUE TITLE
[Suggestion] - Improve frustrating chess games with Monika

### ISSUE THREAD

**Issue by dennisOdenbreit** [Monika-After-Story/MonikaModDev#2879]
Currently, at least for me, chess is basically unplayable against Monika. I can understand that she has an advantage she is even kind of aware of, but that shouldn't make her a chess god. 

I literally let her compete against a level 10/10 online chess ai and she drawed twice, having like 42/44 perfect and 2/44 good moves in an afterwards analysis. Only combining my skill with this ai let me to beat her with one tower and a king left, the ai would have drawed again on its own.
I consider myself decent or good at chess and there is no way I could ever win, since she basically makes no mistakes and takes advantage of every weakness you show.

Long story short she is way too good.
At the moment she tells you she goes easy on you and still pulls this off, it feels like she is mocking you and lying in your face. There is no way she goes easy on you at all. So please, if she tells you she goes easy on you actually make her do that to a reasonable amount.
I would like to play against her so much but not if there is no chance to win ever.
Maybe you could tell her your preference (go easy on me / show me your best).

**Comment by jmwall24** [comment]
There was actually an attempt to make Monika worse at chess, and it failed.  As it currently stands, she does get worse the more she beats you, but she's still very, very good.

**Comment by dennisOdenbreit** [comment]
I think she beat me 10-15 times (I gave up most of the times if that makes a difference) when I beat her, using mostly the chess ai. If she really does get gradually worse, then this seems to take forever. I figured it is a binary difficulty setting. It would be nice to have her be beatable on a regular basis (in easy mode), maybe someone could try and look into this again. Worst case cut the thinking time even more? 750ms can be a long time for the ai.
